app/views/estimate_flow/applicant.html.slim

```diff
@@ -18,35 +18,48 @@
                                           yes_no_options, :first, :last,
                                           legend: { text: t(".over_60.legend") },
                                           hint: { text: t(".over_60.hint") }
-
-    = form.govuk_collection_radio_buttons :dependants,
-            yes_no_options,
-            :first,
-            :last,
-            legend: { text: t(".dependants.legend") },
-            hint: lambda { \
-                            tag.p(t(".dependants.hint")) + \
-                            tag.ul(class: "govuk-list govuk-list--bullet") do \
-                            tag.li(t(".dependants.lives_with")) + \
-                            tag.li(t(".dependants.depends")) \
-                            end + \
-                            tag.div(t(".dependants.no_partner"), class: "govuk-inset-text") \
-                            }
+    - if Flipper.enabled?(:partner)
```

Comment:
Given how gross the hint lambdas are, I think this is as good a way of changing the hint text depending on the feature flag as any. :+1: